README.md

```diff
@@ -26,28 +33,122 @@ There is a large number of templates that are ready to use and are officially su
+| `@asyncapi/html-template`                     | Generates HTML documentation site                               | [click here](https://github.com/asyncapi/html-template)                     |
+| `@asyncapi/markdown-template`                 | Generates documentation in Markdown file                        | [click here](https://github.com/asyncapi/markdown-template)                 |
+| `@asyncapi/ts-nats-template`                  | Generates TypeScript NATS client                                | [click here](https://github.com/asyncapi/ts-nats-template/)                 |
+| `@asyncapi/go-watermill-template`             | Generates Go client using Watermill                             | [click here](https://github.com/asyncapi/go-watermill-template)             |
+| `@asyncapi/dotnet-nats-template`              | Generates .NET C# client using NATS                             | [click here](https://github.com/asyncapi/dotnet-nats-template)              |
+| `@asyncapi/php-template`                      | Generates PHP client using RabbitMQ                             | [click here](https://github.com/asyncapi/php-template)                      |
+| `@asyncapi/dotnet-rabbitmq-template`          | Generates .NET C# client using RabbitMQ                         | [click here](https://github.com/asyncapi/dotnet-rabbitmq-template)          |
 
 <!-- TEMPLATES-LIST:END -->
 
 You can find above templates and the ones provided by the community in **[this list](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate)**
 
+# Generator Filters
+
+This library contains generator filters that can be reused across multiple templates, helping to avoid redundant work. These filters are designed specifically for Nunjucks templates and are included by default with the generator, so there's no need to add then to  dependencies seprately.
+
+This library consists of:
+
+- Custom filters. Check out [API docs](apps/nunjucks-filters/docs/api.md) for complete list
+- Lodash-powered filters. For the list of all available filters check [official docs](https://lodash.com/docs/)
+
 ## Contributing
 
+To Add a major/minor/patch Type Changes:
+
+### Manual
+1.  Create a new markdown file in the `.changeset` directory. The filename should be descriptive of the change.
+2.  Add the following content to the file in this particular format:
+
+```markdown
+---
+"@package-name-1": [type] (major/minor/patch)
+"@package-name-2": [type]
+---
+
+[Provide a brief description of the changes. For example: Added a new Release GitHub Flow to the Turborepo. No new features or bugfixes were introduced.]
+```
+
+#### For Example:
+```markdown
+---
+"@asyncapi/generator": minor
+---
+
+Adding new Release Github Flow to the Turborepo.No new features or bugfixes were introduced.
+
+```
+
+### Using CLI
+```cli 
+npx -p @changesets/cli changeset
+```
+
+3. Simply push the changes with your PR.
```

Comment:
sorry but what does it mean?

using cli I run npm and then push changes and that is it? where is step 1 and 2? what about the description of the release and the scope? CLI will figure it out on its own?

Reply:
I also don't really understand this part. Right now there are three `#` before both Using CLI and Manual, making it seem like "Using CLI" is an alternative to "Manual". However, reading the content, it seems to me that developers should use CLI after they manually add the markdown file in `.changeset`? If my understanding is correct, there should be 4 # before the "Using CLI".

Reply:
no actually this is alternative to manual.

Reply:
@ayushnau remember that if there are 2 folks confused with instructions/code in a similar way, there is no way and you need too address it, improve instructions or add proper code comments for future.

for now I did edit the instructions for you